io-engine/src/core/device_monitor.rs

```diff
@@ -26,26 +45,32 @@ pub async fn device_monitor_loop() {
     loop {
         interval.tick().await;
         if let Some(w) = device_cmd_queue().take() {
-            info!(?w, "executing command");
+            info!("Device monitor exexuting command: {}", w);
```

Comment:
s/exexuting/executing
Again, any reason to remove w from the field names?

Reply:
Reason is poorly formatted, hard-to-follow output.

Reply:
I will fix typo.

Reply:
The possible poor format is only because the way we format the prints. Field names are output using debug/display traits, depending on ? or %.
I'm fine with these changes but we really start loosing the advantages of tracing and every info/debug becomes a simple text/log message

Reply:
I see your point. I'll take a look if it possible to disable field names output in logger formatter.